platforms/common/px4_work_queue/WorkQueueManager.cpp

```diff
@@ -206,9 +206,18 @@ WorkQueueManagerRun(int, char **)
 #ifndef __PX4_QURT
 			const size_t stacksize = math::max(PTHREAD_STACK_MIN, PX4_STACK_ADJUSTED(wq->stacksize));
 #else
-			const size_t stacksize = math::max(8 * 1024, PX4_STACK_ADJUSTED(wq->stacksize));
+            const size_t stacksize = math::max(8 * 1024, PX4_STACK_ADJUSTED(wq->stacksize));
+#endif
+
+#ifndef __PX4_POSIX
+            int ret_setstacksize = pthread_attr_setstacksize(&attr, stacksize);
+#else
+            /* On posix systemm such as MACOS, the desired stacksize round to the nearest multiplier of the system stacksize
+             * It is a requirement of the  pthread_attr_setstacksize* function */
+            const unsigned int page_size = sysconf(_SC_PAGESIZE);
```

Comment:
sysconf(_SC_PAGESIZE) isn't available on QuRT.

Reply:
My bad. This code section was not supposed to run under QuRT